.github/workflows/ci.yml

```diff
@@ -24,12 +24,23 @@ jobs:
         run: sbt "++${{ matrix.scala }} scalafmtCheckAll; scalafmtSbtCheck"
 
       - name: Compile
+        if: startsWith(matrix.scala, '2')
         run: sbt coverage "++${{ matrix.scala }} compile"
 
+      - name: Compile (without coverage)
```

Comment:
This will produce two steps for compile and for tests visible in GitHub Actions, regardless of the Scala version. It's a bit verbose on this config too.

Instead, can we conditionally include `coverage` in the command on the same step? I believe you can do that using Github Actions own Liquid-like syntax and `${{ matrix.scala }}`.

Reply:
I couldn't find a way to do this using GitHub actions' expression syntax, so ended up using bash in order to have a single step.